notebooks/models/Evaluate flair models.ipynb

```diff
@@ -6,7 +6,63 @@
+     "text": [
+      "Kept 194 train samples after removal of non-tagged samples\n",
+      "2020-03-22 17:53:22,532 Reading data from .\n",
+      "2020-03-22 17:53:22,532 Train: flair_train.txt\n",
+      "2020-03-22 17:53:22,532 Dev: flair_test.txt\n",
+      "2020-03-22 17:53:22,532 Test: flair_val.txt\n",
+      "Corpus: 194 train + 59 dev + 28 test sentences\n",
+      "[b'<unk>', b'O', b'B-PERSON', b'I-PERSON', b'B-ORG', b'I-ORG', b'B-GPE', b'I-GPE', b'<START>', b'<STOP>']\n",
+      "2020-03-22 17:53:23,432 https://s3.eu-central-1.amazonaws.com/alan-nlp/resources/embeddings/glove.gensim.vectors.npy not found in cache, downloading to /tmp/tmpxkf2wtcs\n",
+      "  0%|                              | 295936/160000128 [00:01<53:24, 49834.45B/s]^C\n",
+      "Traceback (most recent call last):\n",
+      "  File \"flair_train.py\", line 123, in <module>\n",
+      "    trainer.train(corpus)\n",
+      "  File \"flair_train.py\", line 75, in train\n",
+      "    WordEmbeddings('glove'),\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/flair/embeddings.py\", line 224, in __init__\n",
+      "    cached_path(f\"{old_base_path}glove.gensim.vectors.npy\", cache_dir=cache_dir)\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/flair/file_utils.py\", line 88, in cached_path\n",
+      "    return get_from_cache(url_or_filename, dataset_cache)\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/flair/file_utils.py\", line 181, in get_from_cache\n",
+      "    for chunk in req.iter_content(chunk_size=1024):\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/requests/models.py\", line 750, in generate\n",
+      "    for chunk in self.raw.stream(chunk_size, decode_content=True):\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/urllib3/response.py\", line 496, in stream\n",
+      "    data = self.read(amt=amt, decode_content=decode_content)\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/urllib3/response.py\", line 444, in read\n",
+      "    data = self._fp.read(amt)\n",
+      "  File \"/opt/conda/lib/python3.7/http/client.py\", line 457, in read\n",
+      "    n = self.readinto(b)\n",
+      "  File \"/opt/conda/lib/python3.7/http/client.py\", line 501, in readinto\n",
+      "    n = self.fp.readinto(b)\n",
+      "  File \"/opt/conda/lib/python3.7/socket.py\", line 589, in readinto\n",
+      "    return self._sock.recv_into(b)\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/urllib3/contrib/pyopenssl.py\", line 297, in recv_into\n",
+      "    return self.connection.recv_into(*args, **kwargs)\n",
+      "  File \"/opt/conda/lib/python3.7/site-packages/OpenSSL/SSL.py\", line 1839, in recv_into\n",
+      "    result = _lib.SSL_read(self._ssl, buf, nbytes)\n",
+      "KeyboardInterrupt\n",
+      "  0%|                             | 364544/160000128 [00:01<08:44, 304486.41B/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "! cd ../../models && python flair_train.py"
```

Comment:
Tried this as a single like `python ../../models/flair_train.py` but it had trouble because the python file would look in the wrong directory for the files refered to.

Also had similar problem with `%run ../../models/flair_train.py` type syntax.

I think the `%` type way of calling a python file would be better if we can get it working as the way I'm doing it might not work on Windows.